Task7/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task7
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Вычислите функцию y = f(x)");
+            double x;
+            Console.WriteLine("Введите значение x");
+            x = double.Parse(Console.ReadLine());
+        }
+
+        static double SolutionOfExample(double x)
+        {
+            if (x < -2)
+            {
+                Console.WriteLine($"f(x) = {x / 1}");
+            }
+            else if (x >= -2 && x <= 1)
```

Comment:
Первое неравенство лишнее. Программа перейдет к проверке этого условия только если предыдущее (_x_ < –2) будет ложным.
```suggestion
            else if (x <= 1)
```